File: CPP/Petshop.cpp
#include <iostream>
#include <iomanip>
#include <vector>
using namespace std;

class Petshop
{

private:
    int id;
    string nama_produk;
    double harga_produk;
    int stok_produk;

public:
    Petshop()
    {}

    Petshop(int id, string nama, double harga, int stok)
    {
        this->id = id;
        this->nama_produk = nama;
        this->harga_produk = harga;
        this->stok_produk = stok;
    }

    void set_id(int id)
    {
        this->id = id;
    }

    void set_nama(string nama)
    {
        this->nama_produk = nama;
    }

    void set_harga(double harga)
    {
        this->harga_produk = harga;
    }

    void set_stok(int stok)
    {
        this->stok_produk = stok;
    }

    int get_id()
    {
        return this->id;
    }

    string get_nama()
    {
        return this->nama_produk;
    }

    double get_harga()
    {
        return this->harga_produk;
    }

    int get_stok()
    {
        return this->stok_produk;
    }
    
};

File: JAVA/Petshop.java
public class Petshop {
    private int id;
    private String namaProduk;
    private int hargaProduk;
    private int stokProduk;

    public Petshop() {}

    public Petshop(int id, String nama, int harga, int stok) {
        this.id = id;
        this.namaProduk = nama;
        this.hargaProduk = harga;
        this.stokProduk = stok;
    }

    public void setId(int id) { this.id = id; }
    public void setNama(String nama) { this.namaProduk = nama; }
    public void setHarga(int harga) { this.hargaProduk = harga; }
    public void setStok(int stok) { this.stokProduk = stok; }

    public int getId() { return id; }
    public String getNama() { return namaProduk; }
    public int getHarga() { return hargaProduk; }
    public int getStok() { return stokProduk; }
}
